lib/provider/ui/index.tsx

```diff
@@ -44,17 +44,27 @@ const Main = styled.main`
 
 export function UI({ children }: PropsWithChildren): ReactNode {
   return (
-    <ThemeProvider theme={originalTheme}>
+    <Provider>
+      <InnerUI>
+        {children}
+        <Windows />
```

Comment:
This should have only `{children}`, we should leave the `<Windows />` in the original component.